app/controllers/api_controller.rb

```diff
@@ -21,6 +21,14 @@ def uid
     end
   end
 
+  def userByUid
```

Comment:
In ruby, snake_case is preferred to camelCase. Also, call this twitter_username or something more descriptive for what it does.